src/administrator/components/com_kunena/install/install.script.php

```diff
@@ -17,11 +17,11 @@ class Com_KunenaInstallerScript
 {
 	protected $versions = array(
 		'PHP' => array (
-			'7.0' => '7.0.4',
+			'7.0' => '7.0.3',
 			'5.6' => '5.6.8',
 			'5.5' => '5.5.13',
 			'5.4' => '5.4.13',
-			'0' => '5.4.35' // Preferred version
```

Comment:
You only get this notice when you have <5.3.10 & 5.4.0 < 5.4.13 ect they have all low numbers. So you should not see this error, only on bad hosts, then you need to switch immediately